code/pizzabot.js

```diff
@@ -1,7 +1,89 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+let botSays = document.getElementById('botSays');
+botSays.innerHTML = 'What would you like to order?';
+
+// make inputbox actice (the mouse/cursor is there in beginning)
+document.getElementById("answer").focus();
+
+// used to check if valid pizza is ordered
+let orderedPizza = "";
+
+const isValidPizza = (pizza) => {
+    if (vegetarian.toLowerCase() === pizza.toLowerCase() ||
+        hawaiian.toLowerCase() === pizza.toLowerCase() ||
+        pepperoni.toLowerCase() === pizza.toLowerCase()) {
+        return true
+    } else {
+        return false
+    }
+}
+
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let totalCost = orderQuantity * pizzaPrice
+    return totalCost
+}
+
+const calculateCookingTime = (orderedQuantity) => {
+    if (orderedQuantity < 3) {
+        return 10
+    } else if (orderedQuantity < 6) {
+        return 15
+    } else {
+        return 20
+    }
+}
+
+const submitAnswer = () => {
+
+    // always keep the inputbox active
+    document.getElementById("answer").focus();
+    let answer = document.getElementById("answer").value;
+
+    // If no pizza, or no valid pizza, has been ordered..
+    if (orderedPizza === "") {
+
+        // .. check if the pizza they want is valid
+        let validPizza = isValidPizza(answer)
+
+        if (validPizza) {
+            orderedPizza = answer;
```

Comment:
You seem to be using semicolons only when you want. We have to be consequent - either use them everywhere or don't use them at all